extensions/interactions/ItemSelectionInput/static/item_selection_input.css

```diff
@@ -10,10 +10,14 @@
 
 md-checkbox.item-selection-input-checkbox.md-default-theme .md-icon {
   border-color: #0d48a1;
+  top: 2px;
 }
 md-checkbox.item-selection-input-checkbox.md-default-theme.md-checked .md-icon {
   background-color: rgba(102, 187, 106, 0.87);
 }
+md-checkbox .md-label {
```

Comment:
This rule applies to every md-checkbox .md-label on the entire site. I don't think you want that ...?

Reply:
Sorry for the issue! I forgot adding the ``item-selection-input-checkbox`` class here. I have fixed this now.

Reply:
Sorry for the issue! I forgot adding the ``item-selection-input-checkbox`` class here. I have fixed this now!